rskj-core/src/test/java/co/rsk/peg/BridgeTest.java

```diff
@@ -17,18 +17,26 @@
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
 
 import java.io.IOException;
 import java.math.BigInteger;
 
 import static org.ethereum.config.blockchain.upgrades.ConsensusRule.RSKIP134;
 import static org.ethereum.config.blockchain.upgrades.ConsensusRule.RSKIP143;
+import static org.junit.Assert.fail;
 import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.ArgumentMatchers.anyLong;
 import static org.mockito.ArgumentMatchers.eq;
 import static org.mockito.Mockito.*;
 import static org.mockito.Mockito.mock;
+import static org.powermock.api.mockito.PowerMockito.verifyStatic;
 
+@RunWith(PowerMockRunner.class)
```

Comment:
By definition we discourage using PowerMock. Please rewrite the tests without the need of it.

Reply:
Done!